cli/src/commands/dsl/validate.ts

```diff
@@ -34,6 +35,33 @@ export default class Validate extends BaseCommand {
     };
   }
 
+  private async getTSConfig() {
+    let TSEnvConfig, configFile;
+
+    const EnvTSConfig = glob.sync("**/tsconfig.json")[0];
+
+    this.log(`ENV TS file ${EnvTSConfig}`);
+
+    try {
+      configFile = await fs.readFile(EnvTSConfig);
+    } catch (e) {
+      this.log("Error reading the TypeScript configuration file.");
+    }
+
```

Comment:
If we fail to read the config file do we really need to do the rest of this processing or should we just bail?